tests/unit/moduleapi/propagate.tcl

```diff
@@ -213,6 +213,30 @@ tags "modules" {
                     }
                     close_replication_stream $repl
                 }
+
+                test {module RM_Call of expired key propagation} {
+                    $master debug set-active-expire 0
+
+                    $master setex k1 1 900
+                    wait_for_ofs_sync $master $replica
+                    after 1100
```

Comment:
we can use shorter expire time (`PX 100`), and an `after 110`
and maybe even use a `wait_for_condition` on DEBUG OBJECT to wait for it to expire.
do you think that's worth it?
i'm not sure.

Reply:
mmm sure, why not (for the shorter `after` suggestion)